code/modules/mob/living/carbon/xenomorph/abilities/predalien/predalien_powers.dm

```diff
@@ -34,124 +31,257 @@
+		guttype.targetting = SINGLETARGETGUT
+		to_chat(xeno, SPAN_XENOWARNING("We will now focus our rage on one person!"))
+
+	button.overlays.Cut()
+	button.overlays += image('icons/mob/hud/actions_xeno.dmi', button, action_icon_result)
+	return ..()
 
-	if (ishuman(carbon))
-		var/mob/living/carbon/human/human = carbon
-		human.update_xeno_hostile_hud()
+/datum/action/xeno_action/activable/feral_smash/use_ability(atom/affected_atom)
+	var/mob/living/carbon/xenomorph/predalien_smash = owner
+	var/datum/behavior_delegate/predalien_base/predalienbehavior = predalien_smash.behavior_delegate
 
-	apply_cooldown()
+	if (!action_cooldown_check())
+		if(twitch_message_cooldown < world.time )
+			predalien_smash.visible_message(SPAN_XENOWARNING("[predalien_smash]'s muscles twitch."), SPAN_XENOWARNING("Our claws twitch as we try to grab onto the target but lack the strength. Wait a moment to try again."))
+			twitch_message_cooldown = world.time + 5 SECONDS
+		return //this gives a little feedback on why your lunge didn't hit other than the lunge button going grey. Plus, it might spook marines that almost got lunged if they know why the message appeared, and extra spookiness is always good.
 
-	ADD_TRAIT(xeno, TRAIT_IMMOBILIZED, TRAIT_SOURCE_ABILITY("Devastate"))
-	xeno.anchored = TRUE
+	if (!affected_atom)
+		return
 
-	if (do_after(xeno, activation_delay, INTERRUPT_ALL | BEHAVIOR_IMMOBILE, BUSY_ICON_HOSTILE))
-		xeno.visible_message(SPAN_XENOHIGHDANGER("[xeno] rips open the guts of [carbon]!"), SPAN_XENOHIGHDANGER("We rip open the guts of [carbon]!"))
-		carbon.spawn_gibs()
-		playsound(get_turf(carbon), 'sound/effects/gibbed.ogg', 75, 1)
-		carbon.apply_effect(get_xeno_stun_duration(carbon, 0.5), WEAKEN)
-		carbon.apply_armoured_damage(get_xeno_damage_slash(carbon, base_damage + damage_scale * behavior.kills), ARMOR_MELEE, BRUTE, "chest", 20)
+	if (!isturf(predalien_smash.loc))
+		to_chat(predalien_smash, SPAN_XENOWARNING("We can't lunge from here!"))
+		return
 
-		xeno.animation_attack_on(carbon)
-		xeno.spin_circle()
-		xeno.flick_attack_overlay(carbon, "tail")
+	if (!predalien_smash.check_state() || predalien_smash.agility)
+		return
 
-	playsound(owner, 'sound/voice/predalien_growl.ogg', 75, 0, status = 0)
+	if(predalien_smash.can_not_harm(affected_atom) || !ismob(affected_atom))
+		apply_cooldown_override(click_miss_cooldown)
+		return
 
-	REMOVE_TRAIT(xeno, TRAIT_IMMOBILIZED, TRAIT_SOURCE_ABILITY("Devastate"))
-	xeno.anchored = FALSE
-	unroot_human(carbon, TRAIT_SOURCE_ABILITY("Devastate"))
+	var/mob/living/carbon/carbon = affected_atom
+	if(carbon.stat == DEAD)
+		return
+
+	if (!check_and_use_plasma_owner())
+		return
 
-	xeno.visible_message(SPAN_XENODANGER("[xeno] rapidly slices into [carbon]!"))
+	apply_cooldown()
 
+	predalien_smash.throw_atom(get_step_towards(affected_atom, predalien_smash), grab_range, SPEED_FAST, predalien_smash)
+
+	if (predalien_smash.Adjacent(carbon))
+		predalien_smash.start_pulling(carbon,1)
+		if(isliving(carbon))
```

Comment:
Rather than being here it should be one of the early returns in order to perform this ability. It also probably should have a size check unless you want a predalien able to pick up say the greeno queen and slam her to the ground. If you think its within their power to do so, then I guess its fine as just isliving.

Reply:
i think having an advantage over the enemy hive like this for a "feat" is pretty okay, can be tweaked later on if its busted.